shared/jest.config.js

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line no-undef
+module.exports = {
```

Comment:
I can't see anything in here about matching *.spec.ts files. 
Does this inherit from the base one for 'cli' somehow or is there some smarter way of doing it? 
(I set it up from google first time around, so there probably is!)

Reply:
Earlier in the file there is a `transform` to pass over anything as `.*.ts` to be handled by `ts-jest` which has that configured by default.

Reply:
Ah i see, so you basically just pass it along. Nice